examples/bzlmod_build_file_generation/BUILD.bazel

```diff
@@ -52,6 +52,7 @@ gazelle_python_manifest(
         "//:requirements_windows.txt",
     ],
     tags = ["exclusive"],
+    use_integrity_field = False,
```

Comment:
What is the thinking behind having `use_integrity_field` and `requirements` passed into the macro? IIRC, `requirements` are only used to generate the integrity field. Do you think we get a better API that way?

Reply:
I'd be fine with using the requirements arg as a switch as I am not sure I see a usecase where having two args would be useful. @f0rmiga, let us know if you would like to have this toggled explicitly.

Reply:
I've removed the `use_integrity_field` argument in favor of switching based on whether `requirements` is set.